app/assets/javascripts/browse-columns.js

```diff
@@ -190,6 +191,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     this.updateBreadcrumbs(state)
 
     this.changeColumnVisibility(3)
+    this.showBanner(state.slug)
```

Comment:
This call is in the `showSubsection` function but not the `showSection` function. I don't have a complete picture yet of the intended functionality here but just wanted to check that this was only needed here.

Reply:
Removed the JS - thanks Andy